tests/jballerina-integration-test/src/test/resources/grpc/src/grpcservices/04_client_streaming_service.bal

```diff
@@ -20,32 +20,28 @@ import ballerina/io;
 // Server endpoint configuration
 listener grpc:Listener ep4 = new (9094);
 
-@grpc:ServiceConfig {name:"HelloWorld7",
-    clientStreaming:true}
+@grpc:ServiceConfig {name:"HelloWorld7"}
 @grpc:ServiceDescriptor {
     descriptor: ROOT_DESCRIPTOR_4,
     descMap: getDescriptorMap4()
 }
 service HelloWorld7 on ep4 {
-    resource function onOpen(grpc:Caller caller) {
-        io:println("connected sucessfully.");
-    }
-
-    resource function onMessage(grpc:Caller caller, string name) {
-        io:println("greet received: " + name);
-    }
 
-    resource function onError(grpc:Caller caller, error err) {
-        io:println("Something unexpected happens at server : " + err.reason());
-    }
-
-    resource function onComplete(grpc:Caller caller) {
-        io:println("Server Response");
-        grpc:Error? err = caller->send("Ack");
-        if (err is grpc:Error) {
-            io:println("Error from Connector: " + err.reason());
-        } else {
-            io:println("Server send response : Ack");
+    resource function lotsOfGreetings(grpc:Caller caller, stream<string, error>clientStream) {
+        io:println("connected sucessfully.");
```

Comment:
Shall we use `log` API instead of `io` to print the message?

Reply:
Replaced io with log in both client streaming and bidirectional streaming test cases